### PR TITLE
US22-Task76 Fetch members from project slug

### PR DESCRIPTION
**Pull request by shahih25**
US22Task76 Fetch members from project slug

### CODE REVIEW

**Comment by shahih25** [comment]
Screenshot 2024-02-25 at 2 19 18 PM

File: taigaProject/test/test_main.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from fastapi import FastAPI
```
Comment (shahih25) [inline taigaProject/test/test_main.py:1]: added this test_main file as we are running the test and application from different folders.

Comment (shahih25) [reply taigaProject/test/test_main.py:1]: This main will only be used by the unit tests

File: taigaProject/src/taigaApi/controller/sprint_controller.py [hunk 2]
```diff
@@ -1,12 +1,12 @@
 from fastapi import FastAPI, Request, APIRouter, HTTPException, Header
 from typing import Annotated
 
-from taigaApi.model.projectRequest import ProjectRequest
+from ..model.projectRequest import ProjectRequest
```
Comment (shahih25) [inline taigaProject/src/taigaApi/controller/sprint_controller.py:4]: Relative imports for the unit tests to pass.

**Comment by shahih25** [comment]
Screenshot 2024-02-25 at 12 34 20 PM

**Comment by shahih25** [comment]
Screenshot 2024-02-25 at 2 26 40 PM

**Review by achaur11 — APPROVED** [review]

**Comment by achaur11** [comment]
Looks good to me.

**Pull request merged**